src/python_testing/TC_OpstateCommon.py

```diff
@@ -0,0 +1,1191 @@
+                 TestStep(7, "TH reads from the DUT the CountdownTime attribute"),
+                 TestStep(8, "TH reads from the DUT the PhaseList attribute"),
+                 TestStep(9, "TH reads from the DUT the CurrentPhase attribute"),
+                 TestStep(10, "TH waits for {PIXIT.WAITTIME.COUNTDOWN}"),
+                 TestStep(11, "TH reads from the DUT the CountdownTime attribute"),
+                 TestStep(12, "TH sends Start command to the DUT"),
+                 TestStep(13, "TH sends Stop command to the DUT"),
+                 TestStep(14, "TH reads from the DUT the OperationalState attribute"),
+                 TestStep(15, "TH sends Stop command to the DUT"),
+                 TestStep(16, "Manually put the DUT into a state wherein it cannot receive a Start Command"),
+                 TestStep(17, "TH sends Start command to the DUT")
+                 ]
+        return steps
+
+    async def TEST_TC_OPSTATE_BASE_2_2(self, endpoint=1):
+        cluster = self.test_info.cluster
+        attributes = cluster.Attributes
+        commands = cluster.Commands
+
+        self.init_test()
+
+        asserts.assert_true('PIXIT.WAITTIME.COUNTDOWN' in self.matter_test_config.global_test_params,
+                            "PIXIT.WAITTIME.COUNTDOWN must be included on the command line in "
+                            "the --int-arg flag as PIXIT.WAITTIME.COUNTDOWN:<wait time>")
+
+        wait_time = self.matter_test_config.global_test_params['PIXIT.WAITTIME.COUNTDOWN']
+
+        if wait_time == 0:
+            asserts.fail("PIXIT.WAITTIME.COUNTDOWN shall be higher than 0.")
+
+        # STEP 1: Commission DUT to TH (can be skipped if done in a preceding test)
+        self.step(1)
+
+        # STEP 2: Manually put the DUT into a state wherein it can receive a Start Command
+        self.step(2)
+        self.send_manual_or_pipe_command('{"Name":"OperationalStateChange", "Device":"%s", "Operation":"OnFault", "Param": %s}' %
+                                         (self.device, cluster.Enums.ErrorStateEnum.kNoError))
+        if self.is_ci:
+            time.sleep(1)
+            self.send_manual_or_pipe_command('{"Name":"OperationalStateChange", "Device":"%s", "Operation":"Stop"}' % self.device)
+
+        # STEP 3: TH reads from the DUT the OperationalStateList attribute
+        self.step(3)
+        if self.pics_guard(self.check_pics(("%s.S.A0003" % self.test_info.pics_code))):
+            operational_state_list = await self.read_expect_success(endpoint=endpoint,
+                                                                    attribute=attributes.OperationalStateList)
+
+            operational_state_list_ids = [op_state.operationalStateID for op_state in operational_state_list]
+
+            defined_states = [state.value for state in cluster.Enums.OperationalStateEnum
+                              if state is not cluster.Enums.OperationalStateEnum.kUnknownEnumValue]
```

Comment:
`is not` is for value equality. Be *very careful* and replace all `is not` with `!=` unless the right hand side is NullValue or None. This can be done as a follow-up.

```
>>> a = 1
>>> a is not 1
<stdin>:1: SyntaxWarning: "is not" with a literal. Did you mean "!="?
False
```

Reply:
done